core/txpool/legacypool/legacypool_test.go

```diff
@@ -198,7 +199,7 @@ func validatePoolInternals(pool *LegacyPool) error {
 		if nonce := pool.pendingNonces.get(addr); nonce != last+1 {
 			return fmt.Errorf("pending nonce mismatch: have %v, want %v", nonce, last+1)
 		}
-		if txs.totalcost.Cmp(common.Big0) < 0 {
+		if txs.totalcost.Cmp(common.Uint0) < 0 {
```

Comment:
uint256.Int has `IsZero`, iirc

Reply:
oh btw uint256 cannot be negative, so this whole clause needs to be replace dwith something else.  Probably overflow-check?